extensions/localPublish/src/WebSocketServer.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: javascript convention for classes is camelCase naming for the file (in oppose to Java and c# which is PascalCase)